app/test/ClassTest.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { Coordinate, Time } from '../models/class_defs';
+
+
+describe('Class Tests', () => {
+  describe('#Time', () => {
```

Comment:
The describe should give details on what the test is testing. Example: "#Time: given two equals time with different forms ..."